lib/validators/application_validator.js

```diff
@@ -108,7 +108,9 @@ function checkForValidValues(jdlApplication) {
     'dtoSuffix',
     'entitySuffix',
     'otherModules',
-    'creationTimestamp'
+    'creationTimestamp',
+    'blueprint',
```

Comment:
Blueprint option is deprecated, I think it shouldn’t be added.

Reply:
Ah! I didn't know that. It will be removed in another commit

Reply:
Wait, deprecated or already not supported?

Reply:
Is marked for remove on JHipster 7.
I think we shouldn’t add at all.
blueprint is a single string, blueprints is array.
It doesn’t worth.

Reply:
Alright then!

Reply:
I'm tagging it "to be removed" for JCore 8